### PR TITLE
Fix CVE-2024-24790

### PR DESCRIPTION
**Pull request by FilipeNas**
Bump go version to fix CVE.

Please note I dont know how to work with GO.
I hope this bumps the version.

Referring to this problem:
https://github.com/cri-o/cri-o/issues/8547

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 go 1.22.0
 
-toolchain go1.22.2
+toolchain go1.22.6
```
Comment (kwilczynski) [inline go.mod:3]: This is now how one would upgrade Go version. The above `go` directive would be what you want to change, see:

- [go.mod file reference](https://go.dev/doc/modules/gomod-ref)

**Comment by kwilczynski** [comment]
@saschagrunert, do you think we could move to Go 1.23 already?

**Comment by kwilczynski** [comment]
@FilipeNas, for the future reference. Please do not disregard our Pull Request template; we have it for a reason.

**Comment by saschagrunert** [comment]
> @FilipeNas, for the future reference. Please do not disregard our Pull Request template; we have it for a reason.

this depends a bit if we have go 1.23 in downstream, but I would prefer moving directly to it, yes.

**Comment by kwilczynski** [comment]
> > @saschagrunert, do you think we could move to Go 1.23 already?
> 
> this depends a bit if we have go 1.23 in downstream, but I would prefer moving directly to it, yes.

So, for the time being, the answer is a "no", then. Given that the CVE isn't affecting us, there would be nothing to do here.

**Comment by FilipeNas** [comment]
> > > @saschagrunert, do you think we could move to Go 1.23 already?
> > 
> > 
> > this depends a bit if we have go 1.23 in downstream, but I would prefer moving directly to it, yes.
> 
> @saschagrunert so, for the time being, the answer is a "no", then.
> 
> Given that the CVE isn't affecting us, there would be nothing to do here.

Can't we upgrade to the latest patch version?
I would like for the security scanners to not pick this :))

**Pull request closed without merging**